leaderboard-submissions/metadata/gpt-4o-trileval.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "GPT-4o",
+  "authors": "OpenAI",
+  "url": "https://platform.openai.com/docs/models/gpt-4o",
+  "citation": "OpenAI, 2024",
+  "type": "FOUNDATION",
+  "context": 128000,
+  "is_trained_for_function_calling": true,
+  "details": "gpt-4o-2024-05-13",
+  "closedbook_generations": "results-closedbook-gpt-4o.jsonl",
+  "openbook_generations": "results-trileval-gpt-4o.jsonl",
+  "evidenceprovided_generations": "results-wiki-provided-gpt-4o.jsonl"
```

Comment:
If you do not run certain splits in your eval (i.e., closedbook and evidenceprovided), please set them to `null`. Your `results-trileval-gpt-4o.jsonl` file should also be moved to the `openbook_generations` directory in order for our evaluation script to find it.